### ISSUE TITLE
Blocks page: linked image with no alt text

### ISSUE THREAD

**Issue by joedolson** [WordPress/wporg-main-2022#314]
**Describe the bug**
The "patterns for blocks page" image is linked, but has no alternative text. 

Any image that is the sole content of a link must have alternative text, to provide context for the link. I agree that this image is decorative in character, but because it is also a functional element on the page it needs to be labeled.

I'm not sure that the image actually needs to be linked, however. This can either be solved by removing the link or by adding alternative text e.g. "view large image: description of image"

I think it would probably be better to just remove the link.

**Comment by ryelle** [comment]
> I'm not sure that the image actually needs to be linked, however. This can either be solved by removing the link or by adding alternative text e.g. "view large image: description of image"

Removed the link — it was just the default "link to image" that happens when you insert an image, I removed them from all the others but must've missed this one. This also prompted me to go find where that setting is, and turn it off — so images won't be linked by default now. Thanks!
